F1/F1/Race/Repository/Racing.swift

```diff
@@ -58,11 +58,51 @@ struct RaceInfo: Codable {
         case qualifying = "Qualifying"
         case sprint = "Sprint"
     }
+    init(season: String, round: String, url: String, raceName: String, circuit: Circuit, 
```

Comment:
there should be linting error asking you to please put these one per line if they can't fit onto the line. PLEASE don't ignore the linting warnings.